Signal/src/Jobs/MessageFetcherJob.swift

```diff
@@ -77,25 +75,25 @@ class MessageFetcherJob: NSObject {
 
     private func parseMessagesResponse(responseObject: Any?) -> (envelopes: [OWSSignalServiceProtosEnvelope], more: Bool)? {
         guard let responseObject = responseObject else {
-            Logger.error("\(self.TAG) response object was surpringly nil")
```

Comment:
There's more to do here, but I'm slowly converging on removing the TAGs where possible.